src/Symfony/Component/Yaml/Parser.php

```diff
@@ -368,7 +368,11 @@ private function doParse($value, $flags)
 
                     foreach ($this->lines as $line) {
                         try {
-                            $parsedLine = Inline::parse($line, $flags, $this->refs);
+                            if (isset($line[0]) && in_array($line[0], array('"', "'"), true)) {
```

Comment:
shorter and easier to read imho:
`if (isset($line[0]) && ('"' === $line[0] || "'" === $line[0])) {`